src/pages/Postpaid/PayBill.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { useEffect, useState } from 'react';
+
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import { Styles as CardStyles } from '../../components/Card/styles';
+import { PageBody } from '../../components/PageBody';
+
+import appLogoBig from '../../assets/images/9mobile-logo-big.png';
+import { Column } from '../../components/Column';
+import { Text } from '../../components/Text';
+import { Colors } from '../../themes/colors';
+import { SizedBox } from '../../components/SizedBox';
+import { Card } from '../../components/Card';
+import { Button } from '../../components/Button';
+import { BackButton } from '../../components/BackButton';
+import { convertHexToRGBA } from '../../utils/convertHexToRGBA';
+import { TextField } from '../../components/TextField';
+import { getFieldError } from '../../utils/formikHelper';
+import { Modal } from '../../components/Modal';
+import { SuccessResp } from './Interface';
+import { useLazyFetch, usePost } from '../../customHooks/useRequests';
+import { logger } from '../../utils/logger';
+import { ErrorBox } from '../../components/ErrorBox';
+import { useGlobalStore } from '../../store';
+import { Row } from '../../components/Row';
+import { Spinner } from '../../components/Spinner';
+
+interface CorporateDetailsResp {
```

Comment:
You defined this interface in `src/pages/Postpaid/Interface.ts ` consider importing it instead

Reply:
I'll check this and revert